src/cookbook/ScottPlot.Cookbook/Recipes/Plottable/RadialGauge.cs

```diff
@@ -0,0 +1,325 @@
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.NormBackGauge = true;
+            RadialGauge.StartCap = System.Drawing.Drawing2D.LineCap.Flat;
+            RadialGauge.EndCap = System.Drawing.Drawing2D.LineCap.DiamondAnchor;
+        }
+    }
+
+    public class RadialGaugeStart : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_start";
+        public string Title => "Gauge staring angle";
+        public string Description =>
+            "Angle (in degrees) at which the gauges start: 270 for North (default value), 0 for East, 90 for South, 180 for West, and so on. " +
+            "Expected values in the range [0-360]";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.StartingAngleGauges = 180;
+        }
+    }
+
+    public class RadialGaugeRange : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_range";
+        public string Title => "Gauge angular range";
+        public string Description =>
+            "The maximum angular interval that the gauges will consist of. " +
+            "It takes values in the range [0-360], default value is 360 (full circle).";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.AngleRange = 180;
+        }
+    }
+
+    public class RadialGaugeLabels : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_labels";
+        public string Title => "Gauge labels";
+        public string Description =>
+            "By default, labels are drawn over the gauges.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Palette = ScottPlot.Drawing.Palette.Nord;
+            double[] values = { 100, 80, 65, 45, 20 };
+
+            var RadialGauge = plt.AddRadialGauge(values);
+            RadialGauge.ShowGaugeValues = false;
+        }
+    }
+
+    public class RadialGaugeLabelPos : IRecipe
+    {
+        public string Category => "Plottable: RadialGauge";
+        public string ID => "radialgauge_labelpos";
+        public string Title => "Gauge label position";
+        public string Description =>
+            "The position of the gauges' labels can be adjusted as a percentage of the gauges' angular length. " +
+            "A value of 100 (default) draws the labels at the gauges' end whereas a value of 0 does it at the gauge's beginning.";
```

Comment:
> I wonder if this should be on the interval [0, 1]? It's more common to express fractions this way.

No problem with using the [0, 1]. As a client however, I find it more useful when any percentage property is defined on a [0, 100] interval. If a [0, 1] is considered better, this implies several modifications in `RadialGauglePlot.cs`.

> Also I think the default shouldn't be zero, because the labels look very close to the end. And I think 0 should mean at the base of the gauge, not the tail.

Not sure I get the idea. I believe that a value of 0 draws the label at the beginning (I see that as the gauge's base) and not the ending.

Reply:
I'll refactor this so `GaugeLabelPosition` is a value `[0-1]` representing the position from base to tip